conf/core/ConfigOptions.asciidoc

```diff
@@ -358,39 +358,23 @@ If true, changesets derived can issue delete statements for the reference datase
 passed to the changeset deriver).  If false, no delete statements will be issued for the reference
 dataset.
 
-=== changeset.apidb.size.max
-
-* Data Type: long
-* Default Value: `1000`
-
-This is the maximum number of elements to write to an OSM API database in a changeset. This value
-is used when splitting a changeset into smaller pieces.
-
-NOTE: This is different to `changeset.max.size` which is the maximum number of elements that the
-database can handle in a single changeset.
-
-=== changeset.apidb.writers.max
+=== changeset.apidb.max.writers
 
 * Data Type: long
 * Default Value: `10`
 
 The maximum number of writers to spawn for writing changesets in parallel to an OSM API database.
 
-=== changeset.apidb.writers.throttle
-
-* Data Type: bool
-* Default Value: `false`
-
-Flag to turn on throttling for OSM API changeset writes.  When turned on, each processing thread
-will wait `changeset.api.writers.throttle.time` seconds after a successful write before submitting
-another changeset to the OSM API.
-
-=== changeset.apidb.writers.throttle.time
+=== changeset.apidb.max.size
 
 * Data Type: long
-* Default Value: `30`
+* Default Value: `1000`
 
-The number of seconds after a successful write before submitting another changeset to the OSM API.
+This is the maximum number of elements to write to an OSM API database in a changeset. This value
+is used when splitting a changeset into smaller pieces.
+
+NOTE: This is different to `changeset.max.size` which is the maximum number of elements that the
+database can handle in a single changeset.
```

Comment:
All of the changes above are from a PR that I recently merged into `develop` and need to be restored.  The easiest way would be to grab the latest from develop and then add in your changes only.

Reply:
I had all your changes merged in (OsmApiWriter.* and config and such), but ran into some weird recursive build issue and was unable to build the project (neither locally nor Jenkins). I wanted to make sure my stuff goes through all tests so I reverted it due to lack of time.
When all the other things are addressed I'll merge develop back in and see what's going on.

Reply:
Ok, sounds good.